src/App.js

```diff
@@ -353,14 +355,24 @@ class _AppStackWrapper extends React.Component {
    * These tokens are known as 'registered' tokens
    */
   updateReduxTokens = async () => {
-    const tokens = [];
-    for await (const tokenData of this.props.wallet.storage.getRegisteredTokens()) {
+    if ((!this.props.wallet) || (!this.props.wallet.storage)) {
```

Comment:
This could also benefit from optional chaining
```suggestion
    if (!this.props.wallet?.storage) {
```